api/v1alpha1/bucket_types.go

```diff
@@ -22,6 +22,10 @@ import (
 
 const (
 	BucketFinalizer = "bucket.objectstorage.giantswarm.io"
+	// Secret finalizer need a "/" in the name https://github.com/kubernetes/kubernetes/blob/36981002246682ed7dc4de54ccc2a96c1a0cbbdb/pkg/apis/core/validation/validation.go#L5314-L5320
+	AzureSecretFinalizer = "giantswarm.io/secret.object-storage-operator" // #nosec G101
```

Comment:
This does not work?
```suggestion
	AzureSecretFinalizer = "secrets.objectstorage.giantswarm.io" // #nosec G101
```

Reply:
No, I linked the reason it's not working

Reply:
So why is the one for buckets working?

Reply:
I get the same error than here https://github.com/google/knative-gcp/issues/378
Bucket finalizer is validating by object-storage-operator not core components.

Reply:
I aligned finalizer naming